net/ddns-scripts/files/usr/lib/ddns/update_no-ip_com.sh

```diff
@@ -10,13 +10,15 @@
 # so we send a dummy (localhost) and a seconds later we send the correct IP addr
 #
 local __DUMMY
+local __UPDURL6="http://[USERNAME]:[PASSWORD]@dynupdate6.no-ip.com/nic/update?hostname=[DOMAIN]&myip=[IP]"
```

Comment:
Using uclient-fetch (2020-06-17-c6609861-1)/ libustream-openssl(20200215 2020-03-13-5e1bc342-1) of current master with dynupdate6.no-ip.com fails for me with error 5 (Server hostname does not match SSL certificate).

Switching this dynupdate.no-ip.com seems to be successful.

Reply:
dynupdate.noip.com is ipv4 only, and dynupdate6.noip.com is ipv6 only, just updated PR with the changes.

Reply:
That seems to work, thanks.

```
Tested-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
```